service/pkg.go

```diff
@@ -178,6 +178,7 @@
 package service
 
 import (
+	_ "github.com/databricks/databricks-sdk-go/experimental/mocks" // for mock support
```

Comment:
Do we need this for docs?

Also, can you double check that this include doesn't bloat binary size of e.g. tests?

Reply:
CLI when using SDK without mocks
```
andrew.nester@HFW9Y94129 cli % ls -lh ./cli                                                    
-rwxr-xr-x@ 1 andrew.nester  staff    34M Dec 27 12:50 ./cli
```

CLI when using SDK with mocks and using mocked client in tests
```
andrew.nester@HFW9Y94129 cli % ls -lh ./cli
-rwxr-xr-x@ 1 andrew.nester  staff    34M Dec 27 12:54 ./cli
```